fedimint-core/Cargo.toml

```diff
@@ -48,6 +49,7 @@ miniscript = { version = "9.0.2", features = [ "compiler", "serde" ] }
 secp256k1-zkp = { version = "0.7.0", features = [ "use-serde", "bitcoin_hashes", "global-context" ] }
 macro_rules_attribute = "0.1.3"
 bitvec = "1.0.1"
+parity-scale-codec = { version = "3.5.0", features = ["derive"] }
```

Comment:
Another encoding? Any reason we can't use our built-in encodeable/decodable?

Reply:
unfortunately aleph bft uses it. At least we only use it just in time only for right before a struct is fed into aleph bft.